packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageModule.kt

```diff
@@ -233,24 +148,13 @@ class ExpoImageModule : Module() {
       }
     }
   }
+}
 
-  private fun setBorderRadius(view: ExpoImageViewWrapper, border: String, borderRadius: Float?) {
-    val index = getBorderRadiusIndex(border)
-    val radius = makeYogaUndefinedIfNegative(borderRadius ?: YogaConstants.UNDEFINED)
-    view.setBorderRadius(index, radius)
-  }
-
-  private fun setBorderWidth(view: ExpoImageViewWrapper, borderWidth: String, width: Float?) {
-    val location = getBorderLocationFromBorderWidth(borderWidth)
-    val pixelWidth = makeYogaUndefinedIfNegative(width ?: YogaConstants.UNDEFINED)
-      .ifYogaDefinedUse(PixelUtil::toPixelFromDIP)
-    view.setBorderWidth(location, pixelWidth)
-  }
-
-  private fun setBorderColor(view: ExpoImageViewWrapper, borderColor: String, color: Int?) {
-    val location = getBorderLocationFromBorderColor(borderColor)
-    val rgbComponent = if (color == null) YogaConstants.UNDEFINED else (color and 0x00FFFFFF).toFloat()
-    val alphaComponent = if (color == null) YogaConstants.UNDEFINED else (color ushr 24).toFloat()
-    view.setBorderColor(location, rgbComponent, alphaComponent)
+private inline fun <reified T: View, reified PropType, reified CustomValueType> ViewDefinitionBuilder<T>.PropGroup(
```

Comment:
Could you add a comment that this is temporary and can be removed later in SDK 48?